src/virtio/virtio_endian.h

```diff
@@ -0,0 +1,9 @@
+#pragma once
```

Comment:
Do you mean I should use #ifndef _FOO_H_/#define _FOO_H_ instead?
We still need the protection against double inclusion.

Also, what is the reasoning behind this?

Reply:
Fixed

Reply:
The idea is that, at least within our kernel and little runtime, most files just include a single header (and maybe a few supplemental ones, e.g. for arch or platform specific definitions) that corresponds to the context that it exists in...kernel, process user, etc., and that double inclusions should not happen if we have the headers ironed out properly. Otherwise it's too tempting for a new developer to just throw in arbitrary headers until a build passes. If there's a situation that causes double inclusion, then the headers need sorting out.

@convolvatron might be able to better articulate this...he commented on this in another recent PR...